cmd/share.go

```diff
@@ -63,8 +67,21 @@ var shareCmd = &cobra.Command{
 			fmt.Println("Share revoked for client " + refereeClientID)
 		} else {
 			expiration, _ := cmd.Flags().GetInt64("expiration-seconds")
+			availableAfter := time.Now()
+
+			availableAfterInput, _ := cmd.Flags().GetString("available-after")
+
+			if len(availableAfterInput) > 0 {
+				if aa, err := common.ParseTime(availableAfter, availableAfterInput); err != nil {
+					PrintError(err.Error())
+					os.Exit(1)
+				} else {
+					availableAfter = *aa
+				}
```

Comment:
The `else` is unnecessary here, can do like this:

```go
aa, err := common.ParseTime(...)
if err != nil {
  ...
  os.Exit(1)
  return
}

availableAfter = *aa
```

Reply:
fixed